VideoApi/VideoApi.DAL/Queries/GetConferencesTodayForStaffMemberByHearingVenueNameQuery.cs

```diff
@@ -0,0 +1,46 @@
+using Castle.Core.Internal;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using VideoApi.DAL.Queries.Core;
+using VideoApi.Domain;
+
+namespace VideoApi.DAL.Queries
+{
+    public class GetConferencesTodayForStaffMemberByHearingVenueNameQuery : IQuery
+    {
+        public IEnumerable<string> HearingVenueNames { get; set; }
+    }
+
+    public class GetConferencesTodayForStaffMemberByHearingVenueNameQueryHandler : IQueryHandler<GetConferencesTodayForStaffMemberByHearingVenueNameQuery, List<Conference>>
+    {
+        private readonly VideoApiDbContext _context;
+
+        public GetConferencesTodayForStaffMemberByHearingVenueNameQueryHandler(VideoApiDbContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<List<Conference>> Handle(GetConferencesTodayForStaffMemberByHearingVenueNameQuery query)
+        {
+            var today = DateTime.Today;
```

Comment:
Just a thought does today account for day light saving etc? I thought else where we used UTC.something() to keep it standardised

Reply:
thats a good point, however it appears this has been used everywhere, so if I change it just here that would be in-consistent with the other queries/command we have.

Reply:
I think its good practice to  use utcnow actually, even if its just here I would prefer to change that, let me make this change.

Reply:
oh sorry, just had a look, Datetime.Today just gives you the day, it has nothing to do with the time which is what UTC now will handle when the time change happens basically.